src/app/pages/manage-organisation/findyouradministrator/findyouradministrator.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Rename the file name and folder name to follow the current naming conventions 
ex:- forgot-password-success (include dashes between words)